Resource models with a foreign key

AndrewTelleria opened the pull request:
Fixes issue #115 

Changes: Add two models to the models file in the home app. The first model is a Resource class that has a language field and the second model is a Resources class that pools the Resource class.

divyam3897 commented on home/models.py:
```diff
@@ -1,3 +1,22 @@
+class Resources(models.Model):
+	resource = models.ForeignKey(Resource, on_delete=models.CASCADE)
+	headline = models.CharField(max_length=100)
+	pub_date = models.DateField()
```
pub date not required.

AndrewTelleria replied:
Ok.

divyam3897 commented on home/models.py:
```diff
@@ -1,3 +1,22 @@
+
+class Resources(models.Model):
+	resource = models.ForeignKey(Resource, on_delete=models.CASCADE)
+	headline = models.CharField(max_length=100)
```
give it a better name.

AndrewTelleria replied:
ResourcesPool? Is that a better name?

divyam3897 commented on home/models.py:
```diff
@@ -1,3 +1,22 @@
+	""" A resource that shows students the name of the language, example, and an image. """
+	image = models.ImageField() 	# Might need to install Pillow for ImageField to work.
+	language = models.CharField(max_length=30)
+	example = models.TextField()
```
what is example?

AndrewTelleria replied:
The example is example code for the specified language.

divyam3897 commented:
@AndrewTelleria still on it?

AndrewTelleria commented:
I am still on it. I had to work this weekend and didn't have time.

AndrewTelleria commented:
I have changed the name of the Resources model to ResourcesPool and I removed the pub_date and example. I do not know what else you want to represent with these models.

divyam3897 commented on README.md:
```diff
@@ -97,6 +98,7 @@ After selecting an issue
  5. The pull request will not get merged until and unless the commits are squashed. In case there are multiple commits on the PR, the commit author needs to squash them and not the maintainers cherrypicking and merging squashes.
  6. If the PR is related to any front end change, please attach relevant screenshots in the pull request description
 o
+
```
why these changes?

AndrewTelleria commented:
The PEP8 guidlines are followed, the class names are supposed to have CapWords which they are. If you see anything that is not followed by the PEP8 guidelines please point it out.

AndrewTelleria commented:
As far as squashing the commits I am having trouble. I went to pull from the master branch and it made me make a new commit. When I go to rebase my commit history the last two commits will not show up in my log.

AndrewTelleria commented:
There was a merge conflict that I had to resolve in the README before I could push. I copied and pasted the README from the current master branch into my branch's README hoping that wouldn't have caused any conflict. My branch's README should be identical to the master's README.

divyam3897 commented on home/models.py:
```diff
@@ -1,3 +1,20 @@
+		
+class ResourcesPool(models.Model):
+	resource = models.ForeignKey(Resource, on_delete=models.CASCADE)
+	headline = models.CharField(max_length=100)
```
can you think of a better name then headline?
